lib/hls/hls_parser.js

```diff
@@ -2414,11 +2414,10 @@ shaka.hls.HlsParser = class {
     const contentMimeType = response.headers['content-type'];
 
     if (!contentMimeType) {
-      throw new shaka.util.Error(
-          shaka.util.Error.Severity.CRITICAL,
-          shaka.util.Error.Category.MANIFEST,
-          shaka.util.Error.Code.HLS_COULD_NOT_GUESS_MIME_TYPE,
-          extension);
+      // if the HLS content is lacking in both MIME type metadata and
+      // segment file extensions we fall back to assuming its MP4.
+      const fallbackMimeType = map['notSpecified'];
```

Comment:
It seems to me that you could just use `map['mp4']` instead.

Reply:
Yeah, 🤦🏼 that would make sense.